### PR TITLE
Ports pride pins from TG

### PR DESCRIPTION
**Pull request by nmajask**
# Document the changes in your pull request

Ports ["Adds Pride Pins" #62987](https://github.com/tgstation/tgstation/pull/62987) and ["Lesbian & Intersex Pins, and General Resprite" #65242](https://github.com/tgstation/tgstation/pull/65242) by [axietheaxolotl](https://github.com/axietheaxolotl), also refactors it to have descriptions for each option, though I'm still using placeholders for them right now.

# Spriting
image
image
image

# Wiki Documentation
Add the pin to the vending machine list. 

# Changelog

:cl:  axietheaxolotl
rscadd: Added pride pins
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
just in time

**Comment by JamieD1** [comment]
Do we really want to add this, I can only see issues come from it.

**Comment by stwordcitizen** [comment]
> Do we really want to add this, I can only see issues come from it.

Maybe instead of stopping pride pins you should get rid of people causing issues from seeing pride

**Comment by Smudged** [comment]
> Do we really want to add this, I can only see issues come from it.

People should grow up if a pin is going to cause issues.

**Comment by IsProbablyCatto** [comment]
image
(Its a joke)

**Comment by NCRLouTenant** [comment]
Gotta make ethereals have an NB pin in their survival box at roundstart now

**Review by KillerOrcaCora — APPROVED** [review]
I couldn’t think of a non-offensive joke

**Comment by KillerOrcaCora** [comment]
The Yogstation player base is not mature enough to have these icons which support these expressions they create opportunities to make fun of people which we don't allow here and don't want people to experience, yes we have trashmins but it would create more work for them and it would be better to avoid this altogether.

**Comment by Yarinoi** [comment]
gay 2: gay harder

**Comment by Aquizit** [comment]
oh wait I thought this was Yarinoi's PR

**Comment by nmajask** [comment]
Yogstation jumpscare

**Comment by Yarinoi** [comment]
> Yogstation jumpscare

forever haunted..

**Comment by neeshacark** [comment]
> Yogstation jumpscare


bring it back to us it will get merged if you do it now!!!!!!!!!!

**Comment by Aquizit** [comment]
oops............

**Comment by neeshacark** [comment]
> Yogstation jumpscare

image

🥺

**Comment by nmajask** [comment]
> > Yogstation jumpscare
> 
> bring it back to us it will get merged if you do it now!!!!!!!!!!

Bet
image

**Comment by Aquizit** [comment]
If real I will reopen this again

**Comment by nmajask** [comment]
Real

**Comment by Aquizit** [comment]
REAL

**Review by nmajask — COMMENTED** [review]
real

**Comment by YashinHak** [comment]
i made a thing for this after hearing out some people on what they want changed on it- 
it includes a few more pins and it makes the object sprites a little smaller, i have all the sprites including directional ones for them :3 

now slight issue i have no idea how to actually give these sprites to you

**Comment by nmajask** [comment]
> i made a thing for this after hearing out some people on what they want changed on it- it includes a few more pins and it makes the object sprites a little smaller, i have all the sprites including directional ones for them :3
> 
> now slight issue i have no idea how to actually give these sprites to you

I'm not going to work on it anymore, you can make a PR if its merged to add them or have a maint add it.

**Comment by nmajask** [comment]
@Aquizit are these check fails my fault?

**Comment by Aquizit** [comment]
Nope. Rerun went smoothly.

**Comment by MajManatee** [comment]
I oppose this simply because someone had ":3" in their comment and that made me frown

File: code/modules/clothing/under/accessories.dm [hunk 1]
```diff
@@ -428,3 +428,34 @@
...
+										 "Pansexual Pride"	= list("icon" = "pride_pan",		"info" = "Yellow, pink and blue stripes that represent Pansexual Pride, pride in loving regardless of sex or gender!"),
+										 "Asexual Pride"	= list("icon" = "pride_ace",		"info" = "Black, grey, white and purple stripes that represent Asexual Pride, pride in nonsexual love!"),
+										 "Non-binary Pride"	= list("icon" = "pride_enby",		"info" = "Yellow, white, purple and black stripes that represent Non-Binary pride, pride in identifying as a Non-Binary gender!"),
+										 "Transgender Pride"= list("icon" = "pride_trans",		"info" = "Blue, pink and white stripes that represent Transgender pride, pride in transitioning genders!"),
+										 "Intersex Pride"	= list("icon" = "pride_intersex",	"info" = "A yellow background with a disctint purple circle that represents Intersex pride, pride in atypical sex characteristics!"),
+										 "Lesbian Pride"	= list("icon" = "pride_lesbian", 	"info" = "Orange and pink shaded stripes that represent Lesbian pride, pride in same-gender love among women!"),
+										)
```
Comment (JohnFulpWillard) [inline code/modules/clothing/under/accessories.dm]: [Unintelligible]
```suggestion
	var/static/list/pride_reskins = list(
		"Rainbow Pride" = list(
			"icon" = "pride_rainbow",
			"info" = "A colorful striped rainbow that represents diversity and Gay Pride, pride in same-gender love!"),
		"Bisexual Pride" = list(
			"icon" = "pride_bi",
			"info" = "Purple, dark pink and dark blue stripes that represent Bisexual Pride, pride in loving both binary genders!"),
		"Pansexual Pride" = list(
			"icon" = "pride_pan",
			"info" = "Yellow, pink and blue stripes that represent Pansexual Pride, pride in loving regardless of sex or gender!"),
		"Asexual Pride"	= list(
			"icon" = "pride_ace",
			"info" = "Black, grey, white and purple stripes that represent Asexual Pride, pride in nonsexual love!"),
		"Non-binary Pride" = list(
			"icon" = "pride_enby",
			"info" = "Yellow, white, purple and black stripes that represent Non-Binary pride, pride in identifying as a Non-Binary gender!"),
		"Transgender Pride"= list(
			"icon" = "pride_trans",
			"info" = "Blue, pink and white stripes that represent Transgender pride, pride in transitioning genders!"),
		"Intersex Pride" = list(
			"icon" = "pride_intersex",
			"info" = "A yellow background with a disctint purple circle that represents Intersex pride, pride in atypical sex characteristics!"),
		"Lesbian Pride" = list(
			"icon" = "pride_lesbian",
			"info" = "Orange and pink shaded stripes that represent Lesbian pride, pride in same-gender love among women!"),
		)
```

**Review by JohnFulpWillard — APPROVED** [review]
looks fine, would be even better if you removed the single-letter var from the attack self.

**Review by cuackles — APPROVED** [review]

**Pull request merged**